src/index.html

```diff
@@ -11,7 +11,178 @@
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="form-group">
+          <label for="name">
+            Name:
+            <input
+              type="text"
+              id="name"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="form-group">
+          <label for="age">
+            How old are You?
+            <input
+              type="number"
+              id="age"
+              min="1"
+              max="100"
+              value="12"
+            >
+          </label>
+        </div>
+        <div class="form-group">
+          <label for="date">
+            Full date of birth:
+            <input
+              type="date"
+              id="date"
+            >
+          </label>
+        </div>
+        <div class="form-group">
+          <label for="accept">
+            I accept the term of the agreement
+            <input
+              type="checkbox"
+              id="accept"
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset>
+        <legend>Registration</legend>
+        <div class="form-group">
+          <label for="email">
+            E-mail:
+            <input
+              type="email"
+              id="email"
+              placeholder="email@example.com"
+              required
+            >
+          </label>
+        </div>
+        <div class="form-group">
+          <label for="password">
+            Password:
+            <input
+              type="text"
+              id="password"
+              required
+            >
+          </label>
+        </div>
+      </fieldset>
+      <fieldset>
+        <legend>An interesting fact about you!</legend>
+        <div class="form-group">
+          <label for="cats">
+            Do you love cats?
+            <input
+              type="radio"
+              name="cats"
+              id="yes"
+              value="Yes"
+            >
+            <label for="yes">Yes</label>
+            <input
+              type="radio"
+              name="cats"
+              id="no"
+              value="No"
+            >
+            <label for="no">No</label>
+          </label>
+        </div>
+        <div class="form-group">
+          <label for="color">
+            What is your favorite color?
+            <input
+              type="color"
+              id="color"
+              autocomplete="off"
+            >
+          </label>
+        </div>
+        <div class="form-group">
+          <label for="gotobed">
+            What time do you go to bed?
+            <input
+              type="time"
+              id="gotobed"
+            >
+          </label>
+        </div>
+        <div class="form-group">
+          <label for="cars">
+            What are your favorite brand of cars?
+            <select
+              id="cars"
+              name="cars"
+              multiple
+        >
```

Comment:
closing braket(>) should be on the new line and alined as open tag.  Please improve your code style according this rule
```suggestion
          >
```